src/chrome/content/rules/Freifunk.xml

```diff
@@ -67,6 +71,20 @@
 		to="https://www.youtube.com/user/FREIFUNK2013" />
 	<test url="http://youtube.freifunk.net/?v=RoKQQ5a6E1Q" />
 
+	<!-- Redirect communities to their own domains: -->
+	<rule from="^http://(?:gera|greiz|gera-greiz)\.freifunk\.net/(.*)"
```

Comment:
Please avoid non-capturing groups and add a note that this redirect is required because of a mismatch.